config/routes.rb

```diff
@@ -1,11 +1,6 @@
 Rails.application.routes.draw do
-  root 'pages#hello'
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-  # Reveal health status on /up that returns 200 if the app boots with no exceptions, otherwise 500.
-  # Can be used by load balancers and uptime monitors to verify that the app is live.
-  get "up" => "rails/health#show", as: :rails_health_check
-
-  # Defines the root path route ("/")
-  # root "posts#index"
+  resources :user, only: [:index, :show] do
+    resources :post, only: [:index, :show]
+  end
```

Comment:
- [ ] From convention, these routes should be defined using `users` and `posts`. Plural, not singular. Kindly rename your routes.

  ```
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
  ```